File: Java/CreditCard.java
public class CreditCard extends Payment{
    String bank;
    String dateExpire;
    String cvvNumber;
    Payment id;

    public CreditCard(Integer id, String bank, String dateExpire, String cvvNumber){
        super(id);
        this.bank = bank;
        this.dateExpire = dateExpire;
        this.cvvNumber = cvvNumber;
    }
}

File: Python/creditcard.py
from payment import Payment

class CreditCard(Payment):
    bank = str
    dateExpire = str
    cvvNumber = str

    def __init__(self, id, bank, dateExpire, cvvNumber):
        super().__init__(id)
        self.bank = bank
        self.dateExpire = dateExpire
        self.cvvNumber = cvvNumber